pkg/helpers/describe/projectstatus_test.go

```diff
@@ -8,36 +8,33 @@ import (
 	"text/tabwriter"
 	"time"
 
-	"k8s.io/apimachinery/pkg/api/errors"
-	"k8s.io/apimachinery/pkg/api/meta"
-	"k8s.io/apimachinery/pkg/api/meta/testrestmapper"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/runtime/serializer"
-	utilerrors "k8s.io/apimachinery/pkg/util/errors"
-	fakekubernetes "k8s.io/client-go/kubernetes/fake"
-	kubernetesscheme "k8s.io/client-go/kubernetes/scheme"
-	clientgotesting "k8s.io/client-go/testing"
-	"k8s.io/kubectl/pkg/scheme"
-
 	"github.com/openshift/api"
 	"github.com/openshift/api/annotations"
 	appsv1 "github.com/openshift/api/apps/v1"
 	buildv1 "github.com/openshift/api/build/v1"
 	imagev1 "github.com/openshift/api/image/v1"
-	projectv1 "github.com/openshift/api/project/v1"
 	routev1 "github.com/openshift/api/route/v1"
 	fakeappsclient "github.com/openshift/client-go/apps/clientset/versioned/fake"
 	fakeappsv1client "github.com/openshift/client-go/apps/clientset/versioned/typed/apps/v1/fake"
 	fakebuildclient "github.com/openshift/client-go/build/clientset/versioned/fake"
 	fakebuildv1client "github.com/openshift/client-go/build/clientset/versioned/typed/build/v1/fake"
 	fakeimageclient "github.com/openshift/client-go/image/clientset/versioned/fake"
 	fakeimagev1client "github.com/openshift/client-go/image/clientset/versioned/typed/image/v1/fake"
-	fakeprojectclient "github.com/openshift/client-go/project/clientset/versioned/fake"
-	fakeprojectv1client "github.com/openshift/client-go/project/clientset/versioned/typed/project/v1/fake"
 	fakerouteclient "github.com/openshift/client-go/route/clientset/versioned/fake"
 	fakeroutev1client "github.com/openshift/client-go/route/clientset/versioned/typed/route/v1/fake"
 	osgraph "github.com/openshift/oc/pkg/helpers/graph/genericgraph"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
Nit: there's still a visible value in keeping k8s.io and github.com package imports separated